packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -295,7 +295,6 @@ class AccountSwitcher extends React.Component {
     }
 
     get can_open_multi() {
-        if (this.props.is_eu) return false;
         if (this.props.available_crypto_currencies.length < 1 && !this.props.has_fiat) return true;
         return !this.props.is_virtual;
```

Comment:
Can we destructure`this.props`?

Reply:
@Bahar-FS 
I think it'll be done while refactoring to functional component.

Reply:
fair enough.